.github/workflows/appimage.yml

```
@@ -11,7 +11,7 @@ on:
       - dev
   workflow_dispatch:
 env:
-  publish_pre_dev_labels: '["Beep6581:ladenoise_improv"]'
+  publish_pre_dev_labels: '["Beep6581:whitebalanceopt"]'
```

Remove this before merging:
```suggestion
  publish_pre_dev_labels: '[]'
```

---

OK for me. I change this value to []